easybuild/tools/options.py

```diff
@@ -236,6 +236,8 @@ def config_options(self):
                             'strlist', 'store', ['.git', '.svn']),
             'installpath': ("Install path for software and modules",
                             None, 'store', mk_full_default_path('installpath')),
+            'installpath-modules': ("Install path for modules", None, 'store', None),
```

Comment:
make it clear what the default is (and if this a relative or absolute or both path)